cmd/new-ui/v1beta1/.gitignore

```diff
@@ -0,0 +1 @@
+static
```

Comment:
I thought `static` folder is created under `frontend/static`, is that correct ?

Reply:
Correct. While I was testing my changes around the backend's caching of the static files I had to move them there to test the app locally. I did this in order to check the changes locally and not build a docker image and push it to test the final result.

This was the reason I had add this exclude rule, but if you think it's redundant I can remove it

Reply:
@kimwnasptd But if we follow the README steps to [serve Frontend and Backend](https://github.com/kubeflow/katib/blob/7a2c96f94b07e72c23bdd95350c5cc8fa6ec5fce/pkg/new-ui/v1beta1/README.md#serve-ui-frontend-and-backend), do we need to copy `static` folder from `v1beta1/frontend` to `v1beta1` ?

Reply:
Oh now I think I understand what you mean. These instructions in the readme are for how to serve the frontend **from the backend**, and not a webpack dev server for example.

So then the question becomes do we need to move the files there or can we only use the `--build-dir` arg? 
Is this rephrasing accurate?

Reply:
@kimwnasptd I think setting `--build-dir` args should be enough. Then we don't need to copy `static` folder from the original destination (`v1beta1/frontend`) and we can remove this `.gitigonore` file.

Reply:
Indeed, it should be enough. I'll remove this .gitignore file

Reply:
pushed a commit that removes it